feathr_project/feathr/utils/_envvariableutil.py

```diff
@@ -5,39 +5,101 @@
 from azure.core.exceptions import ResourceNotFoundError
 
 class _EnvVaraibleUtil(object):
-    def __init__(self, config_path):
+    """A utility class to read config variables.
+    If use_env_vars set to False, `get_environment_variable_with_default` will not use os environment variables.
```

Comment:
I've change to have `get` and `get_from_env_or_akv` methods. Also changed the function to preload yaml instead of reading the file for each get operation. Let me know if that looks good.